spec/support/identity_idp_functions.rb

```diff
@@ -1,4 +1,10 @@
 require 'identity-idp-functions'
+require 'identity-idp-functions/proof_address'
+require 'identity-idp-functions/proof_address_mock'
+require 'identity-idp-functions/proof_document'
+require 'identity-idp-functions/proof_document_mock'
+require 'identity-idp-functions/proof_resolution'
+require 'identity-idp-functions/proof_resolution_mock'
```

Comment:
I think we only need the mock ones in specs, we never use the real ones in specs

Reply:
> I think we only need the mock ones in specs, we never use the real ones in specs

Valid point 👍